diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/PackageNaming.kt

```diff
@@ -105,10 +98,12 @@ class PackageNaming(private val configRules: List<RulesConfig>) : Rule("package-
         } else {
             // creating a real package name:
             // 1) getting a path after the base project directory (after "src" directory)
-            // 2) removing src/main/kotlin/java/e.t.c dirs and removing file name
+            // 2) removing src/main/kotlin/java/e.t.c dirs (additional check for multiplatform project layout) and removing file name
             // 3) adding company's domain name at the beginning
             val fileSubDir = filePathParts.subList(filePathParts.lastIndexOf(PACKAGE_PATH_ANCHOR), filePathParts.size - 1)
-                    .dropWhile { LANGUAGE_DIR_NAMES.contains(it) }
+                    .dropWhile {
+                        LANGUAGE_DIR_NAMES.contains(it) || it.endsWith("Main") || it.endsWith("Test")
```

Comment:
let's set it explicitly, unfortunately...

Reply:
I've added a list of currently supported targets for KMP.